### PR TITLE
[ISSUES-4135]fix  hive ds bug

### PR DESCRIPTION
**Pull request by huangKai-2323**
### What is the purpose of the change

Failed to query the hive partition using the hive api

### Related issues/PRs

Related issues: #4135
Related pr:#4133

### Brief change log

- Define the core abstraction and interfaces of the EngineConn Factory;
- Define the core abstraction and interfaces of Executor Manager.

### Checklist

- [x] I have read the [Contributing Guidelines on pull requests](https://github.com/facebook/docusaurus/blob/main/CONTRIBUTING.md#pull-requests).
- [ ] I have explained the need for this PR and the problem it solves
- [ ] I have explained the changes or the new features added to this PR
- [ ] I have added tests corresponding to this change
- [ ] I have updated the documentation to reflect this change
- [ ] I have verified that this change is backward compatible (If not, please discuss on the [Linkis mailing list](https://linkis.apache.org/community/how-to-subscribe) first)
- [ ] **If this is a code change**: I have written unit tests to fully verify the new behavior.

### CODE REVIEW

**Review by casionone — DISMISSED** [review]
LGTM

File: linkis-public-enhancements/linkis-datasource/linkis-metadata-query/server/src/main/java/org/apache/linkis/metadata/query/server/utils/MetadataUtils.java [hunk 1]
```diff
@@ -53,7 +53,7 @@ public class MetadataUtils {
   private static final Logger LOG = LoggerFactory.getLogger(MetadataUtils.class);
 
   public static final String NAME_REGEX =
-      CommonVars.apply("wds.linkis.metadata.query.regex", "^[a-zA-Z\\-\\d_\\.=/:]+$").getValue();
+      CommonVars.apply("wds.linkis.metadata.query.regex", "^[a-zA-Z\\-\\d_\\.=/:,]+$").getValue();
```
Comment (peacewong) [inline linkis-public-enhancements/linkis-datasource/linkis-metadata-query/server/src/main/java/org/apache/linkis/metadata/query/server/utils/MetadataUtils.java:56]: need to remove wds prefix

Comment (huangKai-2323) [reply linkis-public-enhancements/linkis-datasource/linkis-metadata-query/server/src/main/java/org/apache/linkis/metadata/query/server/utils/MetadataUtils.java]: done

**Review by peacewong — APPROVED** [review]
LGTM.

**Pull request merged**